ext/sqlite3/sqlite3.c

```diff
@@ -1277,6 +1277,61 @@ PHP_METHOD(sqlite3, enableExceptions)
 }
 /* }}} */
 
+#if SQLITE_VERSION_NUMBER >= 3006011
+/* {{{ proto bool SQLite3::backup(SQLite3 destination_db[, string source_dbname = "main"[, string destination_dbname = "main"]])
+   Backups the current database to another one. */
+PHP_METHOD(sqlite3, backup)
+{
+	php_sqlite3_db_object *source_obj;
+	php_sqlite3_db_object *destination_obj;
+	char *source_dbname = "main", *destination_dbname = "main";
+	size_t source_dbname_length, destination_dbname_length;
+	zval *source_zval = getThis();
+	zval *destination_zval;
+	zend_error_handling error_handling;
+	sqlite3_backup *dbBackup;
+	int rc; // Return code
+
+	source_obj = Z_SQLITE3_DB_P(source_zval);
+
+	if (zend_parse_parameters_throw(ZEND_NUM_ARGS(), "O|ss", &destination_zval, php_sqlite3_sc_entry, &source_dbname, &source_dbname_length, &destination_dbname, &destination_dbname_length) == FAILURE) {
+		return;
+	}
+
+	destination_obj = Z_SQLITE3_DB_P(destination_zval);
+
+	zend_replace_error_handling(EH_THROW, NULL, &error_handling);
```

Comment:
Same as above.

Reply:
Same. Thanks for taking time to review. Will try to fix that this week.